CouchBaseStorageTests/CouchBaseGrainStorageTests.cs

```diff
@@ -23,7 +23,7 @@ private void AdjustConfig(Orleans.Runtime.Configuration.ClusterConfiguration c)
                 c.Globals.RegisterStorageProvider<Orleans.Storage.OrleansCouchBaseStorage>("Default",
                     new Dictionary<string, string>()
                             {
-                                { "Server","http://localhost:8091" },
+                                { "Servers","http://localhost:8091" },

```

Comment:
Hi @MikeGoldsmith, sorry for the delay in reply, agreed with wanting to avoid a breaking change. 
 I've pushed through a new commit that allows the config to expect either property.  Let me know whether this suits or if you had anything else to raise.